samples/bluetooth/direct_test_mode/src/fem/nrf21540.c

```diff
@@ -282,26 +314,66 @@ static void event_configure(enum nrf21540_trx dir, uint32_t event,
 
 static int gpio_configure(void)
 {
-	int err;
+	int err = 0;
 
+#if DT_NODE_HAS_PROP(NRF21540_NODE, pdn_gpios)
 	/* Configure PDN pin */
+
```

Comment:
Is the blank line needed? I think it slightly  reduces readability.

Reply:
It isn't needed here. I removed it.